selfdrive/car/gm/interface.py

```diff
@@ -140,15 +140,15 @@ def get_params(candidate, fingerprint):
     ret.longPidDeadzoneBP = [0.]
     ret.longPidDeadzoneV = [0.]
 
-    ret.longitudinalKpBP = [5., 35.]
-    ret.longitudinalKpV = [2.4, 1.5]
-    ret.longitudinalKiBP = [0.]
-    ret.longitudinalKiV = [0.36]
+    ret.longitudinalKpBP = [0., 5., 35.]
+    ret.longitudinalKpV = [1.4, 2.425, 2.2]
+    ret.longitudinalKiBP = [0., 35.]
+    ret.longitudinalKiV = [0.33, 0.36]
 
     ret.steerLimitAlert = True
 
     ret.stoppingControl = True
-    ret.startAccel = 0.8
+    ret.startAccel = 0.0
```

Comment:
Together with https://github.com/commaai/openpilot/pull/308, it didn't resume from stop-and-go facing uphill. I wonder it's because of this change (haven't tested with and without changing this line).

Reply:
I would make sure you're testing with all the changes. But I find it hard to believe that it doesn't work. The integrator should wind up, and with the brakes not holding the car back it should take off very reasonably. I run this code as my daily driver.